MadislolincWeb/Pages/AboutUs.razor

```diff
@@ -1,5 +1,11 @@
 @page "/AboutUs"
 
-<PageTitle>Abous us</PageTitle>
+<PageTitle>About sus</PageTitle>
 
-<h1>Meie firma on viimased 10 aastat teeninud kahjumit, sest maksuamet avastab alati, et me tegeleme maksupettusega.</h1>
```

Comment:
This was my fav but oh well

Reply:
it had to be done! otherwise the yfind out we do tax evasion!!!!